beetsplug/synoindex.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+"""Updates the Synology (music) index whenever the beets library is changed.
+
+This assumes beets is being run on Synology DiskStation Manager so synoindex is
+available.  Besides enabling the plugin no configuration is needed.
+"""
+
+from subprocess import run
+
+from beets.plugins import BeetsPlugin
+
+
+class SynoindexPlugin(BeetsPlugin):
+    def __init__(self):
+        super().__init__()
```

Comment:
Can you please switch this to use `super(SynoindexPlugin, self).__init__()` for compatibility?